CHANGELOG.md

```diff
@@ -6,11 +6,162 @@ document.
 
 ## Unreleased / In Rust Nightly
 
-[7c21f91b...master](https://github.com/rust-lang/rust-clippy/compare/7c21f91b...master)
+[d7b5cbf0...master](https://github.com/rust-lang/rust-clippy/compare/d7b5cbf0...master)
+
+## Rust 1.63
+
+Current stable, released 2022-08-11
+
+[7c21f91b...d7b5cbf0](https://github.com/rust-lang/rust-clippy/compare/7c21f91b...d7b5cbf0)
+
+### New Lints
```

Comment:
I think until we have nightly lints, we should check the versions of newly added lints when we write the changelog (That should also be part of the docs IIRC). But I wouldn't hold the PR, this can be done in a followup :upside_down_face:

Reply:
Done in #9326